tools/local-dev.sh

```diff
@@ -6,27 +6,43 @@ set -e
 ## Usage: source tools/local-dev.sh
```

Comment:
Please include a comment here about the env var, similar to the one in `install.sh`